app/services/bulk_api_import/fhir_condition_importer.rb

```diff
@@ -27,7 +27,7 @@ def metadata
   def build_attributes
     {
       id: translate_id(@resource.dig(:identifier, 0, :value), org_id: @organization_id),
-      patient_id: translate_patient_id(@resource[:subject][:identifier], org_id: @organization_id),
+      patient_id: translate_id(@resource[:subject][:identifier], org_id: @organization_id),
```

Comment:
Here and elsewhere: We want to translate the patient_id to the ID of the Patient and not the ID of its PatientBusinessIdentifier. This was the source of the bug.